package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "mapic.js",
+  "version": "1.3.8",
+  "description": "![Mapic Logo](https://cloud.githubusercontent.com/assets/2197944/19607635/5c434458-97cb-11e6-941b-e74e83b385ba.png) # Mapic.js client library This is the open source client library front-end for Mapic.",  
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "test": "karma start"
+  },
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "karma": "^1.3.0",
+    "karma-chai": "^0.1.0",
+    "karma-chrome-launcher": "^2.0.0",
+    "karma-mocha": "^1.2.0",
+    "mocha": "^3.1.2"
+  },
+  "author": "Mapic",
```

Comment:
You can add these "contributors" instead of "author", and include yourself https://github.com/mapic/engine/blob/master/package.json#L23